openslides_backend/action/actions/user/update.py

```diff
@@ -84,18 +84,3 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             self.check_limit_of_user(1)
 
         return instance
-
-    def apply_instance(
```

Comment:
This looks like it was missed during a rebase, but it seems no longer necessary - at least the test introduced in https://github.com/OpenSlides/openslides-backend/pull/1664 together with this change works without this code, so I removed it. Maybe you have more insight about this @r-peschke

Reply:
seems okay